src/libs/actions/Session/index.js

```diff
@@ -138,44 +103,41 @@ function resendValidationLink(login = credentials.login) {
  * @param {String} login
  */
 function fetchAccountDetails(login) {
```

Comment:
Should we rename this to `beginSignIn` to match the API command?

Reply:
+1